app/routes/index.js

```diff
@@ -21,7 +21,6 @@ export default class IndexRoute extends Route {
       }
     } catch (error) {
       console.error(error.message);
-      alert('Failed');
```

Comment:
Why did we remove this?

Reply:
When fetch request failed, I was encountering two alerts. I thought one will do the job. That's why removed it here

Reply:
What should I do ?? Include it or remove it ??

Reply:
Remove it for now. We need to get rid of this API call as you had rightly pointed out in discord :)
Having 2 API calls on the homepage

Reply:
Ok, resolving the conversation